src/Database/Entities/Provider.php

```diff
@@ -80,4 +81,27 @@ public function setName(string $name): void
     {
         $this->name = $name;
     }
+
+    /**
+     * Serialize entity as an array
+     *
+     * @return mixed[]
+     */
+    public function toArray(): array
```

Comment:
If there are no controllers for Read/List of Providers, do we need toArray?

Reply:
@edwardmurrell-loyaltycorp the `toArray` method is required as part of the extension of `Entity`.